qemu/tests/win_heavyload.py

```diff
@@ -129,7 +142,7 @@ def add_option(cmd, key, val):
 
         error_context.context("Verify vm is alive", logging.info)
         utils_misc.wait_for(vm.verify_alive,
-                            timeout=test_timeout, step=steping)
+                            timeout=test_timeout + 2, step=steping)
```

Comment:
Why plus 2 here?  It seem no difference between trying 11 times and 10 times.

Reply:
timeout for func wait_for() is same with the "DURATION" for heavyload.  It need few second to start and exit the progress. i want to extend 2 second here. 
Take account of "step=60" in wait_for(), use time.sleep(2) instead, thanks !